contracts-periphery/src/test/UmbraBatchSend.t.sol

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "./utils/DSTestPlus.sol";
+import "../UmbraBatchSend.sol";
+
+interface UmbraToll {
+  function toll() external returns(uint256);
+}
+
+abstract contract UmbraBatchSendTest is DSTestPlus {
+
+  UmbraBatchSend router;
+  MockERC20 token;
+
+  address constant umbra = 0xFb2dc580Eed955B528407b4d36FfaFe3da685401;
+  address tokenAddr;
+  address alice = address(0x202204);
+  address bob = address(0x202205);
+
+  uint256 alicePrevBal;
+  uint256 bobPrevBal;
+  uint256 umbraPrevBal;
```

Comment:
These are never initialized and therefore are always zero—do we need these variables or should they be removed?